### PR TITLE
Splashscreen: Show ThirdPartyLibraries.html at About/Libraries - Fixes #17054

### LINKED ISSUE #17054: Hyperlinks in About > Libraries open empty tabs in the application
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

The hyperlinks in the about dialog open empty tabs in the FreeCAD application, instead of the system's web browser.
I guess this is due to the removal of the web features @chennes ? Are there other locations which need a workaround?
grafik

### Full version info

```shell
OS: Windows 11 build 26100
Word size of FreeCAD: 64-bit
Version: 1.1.0dev.38871 (Git)
Build type: Release
Branch: main
Hash: 813ee36cb80e0d29c2c9b991a7b2fee2c61f5cf6
Python 3.11.9, Qt 5.15.13, Coin 4.0.3, Vtk 9.2.6, OCC 7.7.2
Locale: German/Germany (de_DE)
Stylesheet/Theme/QtStyle: FreeCAD Dark.qss/FreeCAD Dark/Fusion
Installed mods: 
  * CfdOF 1.27.10
  * Curves 0.6.48
  * dodo 1.0.1
  * fasteners 0.5.29
  * fasteners.backup1721936159.7542229 0.5.24 (Disabled)
  * freecad.gears 1.3.0
  * Manipulator 1.5.7
  * OpenTheme 2024.9.1
  * OpticsWorkbench 1.0.24
  * Rocket 3.3.0
  * sheetmetal 0.4.26
  * Silk 0.1.5
  * Silk.backup1727166062.7227929 0.1.5 (Disabled)
```

### Subproject(s) affected?

Core

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by NewStorm**
Currently, the 3rd party libraries information is contained in the ThirdPartyLibraries.html and get additionally generated in the About/Libraries Tab. 
With this pull request, the version strings get injected into the html via cmake and instead of generating the libraries listing, the html gets shown instead.
This removes redundancies, duplicate source code and splits data from logic.
It also fixes #17054

### CODE REVIEW

File: src/Doc/ThirdPartyLibraries.html.cmake [hunk 1]
```diff
@@ -56,36 +60,33 @@ <h3><a name="_TocKDL"></a>KDL</h3>
 <p>License: LGPL version 2.1: <a href="#_LGPLv21">LGPL v2.1</a></p>
 
 
-<hr>
-
-<h3><a name="_TocLibArea"></a>libarea</h3>
-<p>Web site: <a href="https://github.com/danielfalck/libarea/">https://github.com/danielfalck/libarea/</a></p>
-<p>License: BSD-3-clause: <a href="https://github.com/danielfalck/libarea/blob/master/License.txt">https://github.com/danielfalck/libarea/blob/master/License.txt</a></p>
-
-
```
Comment (luzpaz) [inline src/Doc/ThirdPartyLibraries.html.cmake:65]: looks like libarea was removed but AFAICT not mentioned anywhere else?

Comment (NewStorm) [reply src/Doc/ThirdPartyLibraries.html.cmake:65]: According to slpitronics post in [this forum thread](https://forum.freecad.org/viewtopic.php?style=4&t=43857), we are using our own copy which is diverting from the upstream source. I can pack it back inside again, but it looks like the 3rd Party Library overviews on the web and everywhere else are a bit outdated. 
Should I add it back again (without version string) ?

Comment (luzpaz) [reply src/Doc/ThirdPartyLibraries.html.cmake:65]: Unsure. Let's ping @sliptonic

Comment (sliptonic) [reply src/Doc/ThirdPartyLibraries.html.cmake:65]: Let's add it back without version.

**Comment by chennes** [comment]
This is a much nicer way of displaying this information, thanks for the PR.

**Comment by chennes** [comment]
Backported to 1.0

**Pull request merged**